iXBRLViewerPlugin/viewer/src/js/inspector.js

```diff
@@ -299,7 +324,7 @@ Inspector.prototype._anchorList = function (fact, anchors) {
         }
     }
     else {
-        $("<li><i>None</i></li>").appendTo(html);
+        $(`<li><i>${i18next.t("common.none")}</i></li>`).appendTo(html);
```

Comment:
It seems like we should avoid interpolation of any kind into HTML strings.  Can we either use `.text` (on the inner `<i>`) or use `data-i18n` and `.localize()`?

Reply:
Fixed in fcaaa71 (with `.text`)